packages/mobile/src/pincode/PincodeEnter.tsx

```diff
@@ -31,8 +31,6 @@ interface StateProps {
 type Props = StateProps & WithTranslation & StackScreenProps<StackParamList, Screens.PincodeEnter>
 
 class PincodeEnter extends React.Component<Props, State> {
-  static navigationOptions = { gestureEnabled: false, ...nuxNavigationOptions }
-
```

Comment:
it's important that the user can't swipe to go back, the gesture part should remain.

Reply:
I don't think this was being used anymore, this was used in a previous version of `react-navigation`, now the options have to be set wherever the navigation is defined (`Navigator` file in our case). In fact, I'm able to swipe back on the Testflight build. I added the `gestureEnabled: false` in the options of this screen so as to disallow that :)

Reply:
Yes I see what you mean. 
The code is currently in a transition phase for screen options as we decided to relocate them to each screen file using the `navigationOptions` static prop on the component, like it was with `react-navigation` v4.
We decided this to avoid cluttering the `Navigator` as we felt it belonged better with the component.